main.c

```diff
@@ -1350,6 +1355,10 @@ registerimage(const char * region, const char * snapshot, const char * name,
 		goto err1;
 	if ((archenc = rfc3986_encode(arch)) == NULL)
 		goto err2;
+    
+    /* Get image size in GB and set to 10 if less than 10. */
+    size_t gb_size = (*imgsz)/(1024 * 1024 * 1024);
```

Comment:
Put variable declarations at the top of the function.

Also, the indentation is wrong.

Also, there should be whitespace on either side of the `/` character.

Also, `if(gb_size < 10) gb_size = 10;` should be
```
if (gb_size < 10)
        gb_size = 10;
````